setup.py

```diff
@@ -174,7 +174,8 @@ def load_description():
     entry_points={
         'console_scripts': [
             'falcon-bench = falcon.cmd.bench:main',
-            'falcon-print-routes = falcon.cmd.print_routes:main'
+            'falcon-inspect-app = falcon.cmd.inspect_app:main',
+            'falcon-print-routes = falcon.cmd.inspect_app:main',
```

Comment:
It would be nice if the app would print out a deprecation message when using the old script name.

Reply:
Yes I was also thinking about this. Maybe it would also select the "routes only switch by default"?